Makefile

```diff
@@ -178,6 +198,8 @@ release: tag_major_minor
 	docker push $(NAME)/node-base:$(MAJOR)
 	docker push $(NAME)/node-chrome:$(MAJOR)
 	docker push $(NAME)/node-firefox:$(MAJOR)
+	docker push $(NAME)/node-opera:$(MAJOR)
+	docker push $(NAME)/node-phantomjs:$(MAJOR)
```

Comment:
Why is `node-phantomjs` here? This image was deprecated a while ago.

Reply:
Deleted.